docs/IRISMapCube.rst

```diff
@@ -0,0 +1,155 @@
+
+Dimensions
+----------
+
+As the IRISMapCube object inherits from NDCube_, we can use the two properties of NDCube_
+which allow us to the data shape and the axis types of our IRISMapCube object. These
+properties are described in the NDCube.Dimensions_ section.
+
+Cropping and Indexing
+---------------------
+
+One of the most powerful capability of IRISMapCube, coming from NDCube_, is the slicing
+process. To slice the cube, we can slice the IRISMapCube with an Array-like Indexing or
+we can crop it by the Real World Coordinates. As the IRISMapCube object inherits from
+NDCube_, we can use the described processes in the NDCube.Slicing_ section.
+
+Data Manipulation
+-----------------
+
+Now we have our IRISMapCube object and we know how access to all the information it contains,
+we can manipulate the data with the below presented methods.
+
+Exposure Time Correction
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can apply the exposure time correction to data and to the uncertainty and
+this method adjusts the unit: ::
+
+    >>> my_cube.apply_exposure_time_correction()
+
+We also can undo the exposure time correction by doing: ::
+
+    >>> my_cube.apply_exposure_time_correction(undo=True)
+
+The correction is only applied (undone) if the object's unit doesn't (does) already
+include inverse time. This can be overridden so that correction is applied (undone)
+regardless of unit by setting ``force=True``. Use one of the two lines above to apply
+(undone) by using the force kwarg: ::
+
+    >>> my_cube.apply_exposure_time_correction(force=True)
+    >>> my_cube.apply_exposure_time_correction(undo=True, force=True)
+
+Dust particle mask
+^^^^^^^^^^^^^^^^^^
+
+If we take time to look inside the data, we can see that some spots are not relevant with
+the data we want to study. These spots are dust particles that can come from space or a
+not perfect construction of the satellite. In IRISMapCube, we can use a method that will
+modify the mask of our object with the dust particles positions. We can use this method
+by doing: ::
+
+    >>> my_cube.apply_dust_mask()
+
+Now, our ``my_cube.mask`` contains the dust particles positions and we can use it to
+select only the data we want. If we want to remove the dust particles positions from
+our mask, we can call again this method with the ``undo`` kwarg. ::
+
+    >>> my_cube.apply_dust_mask(undo=True)
+
+If we don't remember or we don't know if the dust particles positions are applied or not
+in our ``my_cube.mask``, we can check an attribute of our object. ::
+
+    >>> my_cube.dust_masked
+
+If the result is ``True``, the dust particles positions are applied in our ``my_cube.mask``.
+If the result is ``False``, the dust particles positions are not applied.
+
+Visualization
+-------------
+
+There is two different ways to visualize our IRISMapCube object. The first one is to use
+the representation of our object and the second one is to plot the data of our object.
+
+Representation
```

Comment:
I don't think this should be its own section, but should be included as part of the ```Initialization``` section.  It's not really visualization.  It's inspection.

Reply:
Moved